vector/src/main/java/im/vector/app/features/home/room/detail/AutoCompleter.kt

```diff
@@ -182,7 +182,7 @@ class AutoCompleter @AssistedInject constructor(
                 .with(object : AutocompleteCallback<String> {
                     override fun onPopupItemClicked(editable: Editable, item: String): Boolean {
                         // Detect last ":" and remove it
-                        var startIndex = editable.lastIndexOf(":")
+                        var startIndex = editable.subSequence(0, editText.selectionStart).lastIndexOf(":")
```

Comment:
Maybe update the comment above?

Reply:
https://github.com/vector-im/element-android/pull/4566/files#diff-b7f049abc21b0d7948c0e51a423be888d10aa33cf02111be634b4cd4f741cf8b